src/main/java/me/jellysquid/mods/sodium/client/model/light/smooth/SmoothLightPipeline.java

```diff
@@ -88,9 +89,16 @@ private void applySidedBrightness(QuadLightData out, Direction face, boolean sha
         float brightness = this.lightCache.getWorld().getBrightness(face, shade);
         float[] br = out.br;
 
-        for (int i = 0; i < br.length; i++) {
-            br[i] *= brightness;
-        }
+//        for (int i = 0; i < br.length; i++) {
+//            br[i] *= brightness;
+//        }
+
+        // What's the use of this? br is never accessed after multiplying it by brightness.
```

Comment:
Originally, `br` is modified in-place as `out.br`, which I assume is used later from something accessing `QuadLightData` (i.e. the renderers themselves).
Your code should instead assign `out.br` to preserve this behavior.

Reply:
out.br is final though